src/test/groovy/org/prebid/server/functional/tests/FilterMultiFormatSpec.groovy

```diff
@@ -0,0 +1,383 @@
+            imp[0].banner = null
+            imp[0].audio = Audio.defaultAudio
+        }
+
+        and: "Account in the DB with preferred media type"
+        def accountConfig = new AccountAuctionConfig(preferredMediaType: [(BidderName.GENERIC): BANNER])
+        def account = new Account(uuid: bidRequest.accountId, config: new AccountConfig(auction: accountConfig))
+        accountDao.save(account)
+
+        when: "PBS processes auction request"
+        def bidResponse = pbsService.sendAuctionRequest(bidRequest)
+
+        then: "PBS should make bidder request"
+        def bidderRequests = bidder.getBidderRequests(bidRequest.id)
+        assert bidderRequests.imp.audio
+
+        and: "Bid response shouldn't contain warning"
+        assert !bidResponse.ext.warnings
+    }
+
+    def "PBS shouldn't respond with warning and make a bidder call when request doesn't contain multi format and preferred media type specified at request level"() {
+        given: "PBS with adapter configuration"
+        def pbsService = pbsServiceFactory.getService(
+                "adapters.generic.ortb.multiformat-supported": "false")
+
+        and: "Default bid request with audio and nativeObj type and preferredMediaType BANNER"
+        def bidRequest = BidRequest.defaultBidRequest.tap {
+            imp[0].banner = null
+            imp[0].audio = Audio.defaultAudio
+            ext.prebid.bidders = new PreferredBidders(generic: new GenericPreferredBidder(preferredMediaType: BANNER))
+        }
+
+        when: "PBS processes auction request"
+        def bidResponse = pbsService.sendAuctionRequest(bidRequest)
+
+        then: "PBS should make bidder request"
+        def bidderRequests = bidder.getBidderRequests(bidRequest.id)
+        assert bidderRequests.imp.audio
+
+        and: "Bid response shouldn't contain warning"
+        assert !bidResponse.ext.warnings
+    }
+
+    def "PBS shouldn't respond with warning and make a bidder call when request doesn't contain multi format and multi format is false and preferred media type specified at request level with null"() {
+        given: "PBS with adapter configuration"
+        def pbsService = pbsServiceFactory.getService(
+                "adapters.generic.ortb.multiformat-supported": "false")
+
+        and: "Default bid request with banner and audio type"
+        def bidRequest = BidRequest.defaultBidRequest.tap {
+            imp[0].banner = Banner.getDefaultBanner()
+            imp[0].audio = Audio.defaultAudio
+            ext.prebid.bidders = new PreferredBidders(generic: new GenericPreferredBidder(preferredMediaType: NULL))
+        }
+
+        when: "PBS processes auction request"
+        def bidResponse = pbsService.sendAuctionRequest(bidRequest)
+
+        then: "PBS should make bidder request"
+        assert bidder.getBidderRequests(bidRequest.id)
+
+        and: "Bid response shouldn't contain warning"
+        assert !bidResponse.ext?.warnings
```

Comment:
```
If the request does not contain a preferred type:
The bidder will not be called and a warning will be returned.
```
https://github.com/prebid/prebid-server/issues/2711#issue-1677787288

`if a preferred mediatype is not specified, there's no default. The bid adapter may choke.`

https://github.com/prebid/prebid-server/issues/2711#issuecomment-1518069774